app/src/main/java/org/simple/clinic/editpatient/EditPatientUi.kt

```diff
@@ -29,4 +29,5 @@ interface EditPatientUi {
   fun showProgress()
   fun hideProgress()
   fun setupUi(inputFields: InputFields)
+  fun showColonyOrVillagesList(colonyOrVillageList: List<String>)
```

Comment:
Not needed in this PR, but let's rename this method to `setColonyOrVillagesList` or `setColonyOrVillagesAutoComplete` since we are not actually showing once they are loaded.WDYT?